eclide/RepositoryFrame.h

```diff
@@ -60,8 +60,6 @@ class CRepositoryFrameT :	public CRepositorySlotImpl,
     {
         T * pT = static_cast<T*>(this);
         ContextState state;
-        if (s.mods.size() + s.attrs.size() == 0)
-            return;
```

Comment:
I think this needs some additional tests:
1 - Is local or remote?
2 - Is the blue folder readonly (plugins, bundles and stdlib)